docs/getting_started.qmd

```diff
@@ -2,129 +2,21 @@
 
-## Example: draw all venues
+## What is the output generated by SPC?
 
-[This
-script](https://github.com/alan-turing-institute/uatk-spc/blob/main/python/draw_venues.py)
-reads a protobuf file, then draws a dot for every venue, color-coded by
-activity.
-
-![](img/venues.gif)
-
-## Understand the schema
-
-Here are some helpful tips for understanding the
-[schema](https://github.com/alan-turing-institute/uatk-spc/blob/main/synthpop.proto).
-
-Each .pb file contains exactly one `Population` message. In contrast to
-datasets consisting of multiple `.csv` files, just a single file contains
-everything. Some of the fields in `Population` are lists (of people and
-households) or maps (of venues keyed by activity, or of MSOAs). Unlike a flat
-`.csv` table, there may be more lists embedded later. Each `Household` has a
-list of `members`, for example.
-
-The different objects refer to each other, forming a graph structure. The
-protobuf uses `uint64` IDs to index into other lists. For example, if some
-household has `members = [3, 10]`, then those two people can be found at
-`population.people[3]` and `population.people[10]`. Each of them will have the
-same `household` ID, pointing back to something in the `population.households`
-list.
-
-### Flows
-
-SPC models daily travel behavior of people as "flows." Flows are broken down by
-by an
-[activity](https://github.com/alan-turing-institute/uatk-spc/blob/feffb857c6fbbd7c2b3dd1f3cb46e67677185897/synthpop.proto#L196)
--- shopping/retail, attending primary or secondary school, working, or staying
-at home. For each activity type, a person has a list of venues where they may
-do that activity, weighted by a probability of going to that particular venue.
-
-Note that `flows_per_activity` is stored in `InfoPerMSOA`, not `Person`. The
-flows for retail and school are only known at the MSOA level, not individually.
-So given a particular `Person` object, you first look up their household's MSOA
--- `msoa = population.households[ person.household ].msoa` and then look up
-flows for that MSOA -- `population.info_per_msoa[msoa].flows_per_activity`.
-
-Each person has exactly 1 flow for home -- it's just `person.household` with
-probability 1. A person has 0 or 1 flows to work, based on the value of
-`person.workplace`.
+SPC creates Google's [protocol buffer files](https://developers.google.com/protocol-buffers/docs/overview) a platform-neutral, extensible mechanism for serializing structured data in bi-directional way (forward-compatible and backward-compatible). SPC encodes the enriched synthetic population [schema](https://github.com/alan-turing-institute/uatk-spc/blob/main/synthpop.proto) in .proto files (.pb). You can read the "protobuf" (shorthand for a protocol buffer file) in any
```

Comment:
From the outputs page: "Over time, we may add more data to SPC or change the schema. Protocol buffers are designed to let combinations of new/old code and data files work together, but we don’t intend to use this feature. We may make breaking changes, like deleting fields. We’ll release a new version of the schema and output data every time and document it here. You should depend on a specific version of the data output in your code, so new releases don’t affect you until you decide to update."

The forward/backward compatible aspect of protobufs is explicitly not used in this project, I wouldn't mention it at all

Reply:
Ok